hw_diag/utilities/network_watchdog.py

```diff
@@ -133,17 +117,10 @@ def ensure_network_connection(self) -> None:
         self.LOGGER.warning(
             f"Network is not connected! Lost connectivity count={self.lost_count}")
 
-        if self.lost_count >= self.NM_RESTART_THRESHOLD:
-            self.LOGGER.warning(
-                "Reached threshold for nm restart for recovering network.")
-            self.restart_network_manager()
```

Comment:
why did we get rid of network manager restart? Wouldn't this push our action threshold to 3 hours for non connectivity.

Reply:
ok, found it. Ignore the comment.